src/modules/mc_pos_control/mc_pos_control_params.c

```diff
@@ -351,12 +354,11 @@ PARAM_DEFINE_FLOAT(MPC_XY_ERR_MAX, 2.0f);
 /**
  * Maximum horizontal velocity setpoint in Position mode
  *
- * If velocity setpoint larger than MPC_XY_VEL_MAX is set, then
- * the setpoint will be capped to MPC_XY_VEL_MAX
- *
  * The maximum sideways and backward speed can be set differently
  * using MPC_VEL_MAN_SIDE and MPC_VEL_MAN_BACK, respectively.
  *
+ * Set lower than MPC_XY_VEL_MAX
```

Comment:
It can actually be equal to `MPC_XY_VEL_MAX`

Reply:
But then there's an artificial limitation in the background. Anyways the user maybe doesn't need to care. I'll adapt the description 👍